aws/templates/id/id_cognito.ftl

```diff
@@ -90,33 +92,37 @@
                 "Default" : 30
             },
             {
-                "Name" : "allowUnauthIds",
+                "Name" : "allowUnauthenticatedIds",
                 "Default" : false
             }
             {
                 "Name" : "passwordPolicy",
                 "Children" : [
                     {
-                       "Name" : "MinimumLength",
-                       "Default" : "8"
+                       "Name" : "minimumLength",
+                       "Default" : "10"
                     },
                     {
-                        "Name" : "Lowercase",
+                        "Name" : "lowercase",
                         "Default" : true
                     },
                     {
-                        "Name" : "Uppercase",
+                        "Name" : "uppercase",
                         "Default" : true
                     },
                     {
-                        "Name" : "Numbers",
+                        "Name" : "numbers",
                         "Default" : true
                     },
                     {
-                        "Name" : "SpecialCharacters",
-                        "Default" : false
+                        "Name" : "specialCharacters",
+                        "Default" : true
                     }
                 ] 
+            },
+            {
+                "Name" : "Links",
+                "Default" : {}
```

Comment:
if we merge #175, this shuld be changed to use the explicit config for children with repeating content